docs/tutorials/pytorch-export.ipynb

```diff
@@ -123,24 +135,11 @@
         "id": "2Ujt2OjtpERw"
       },
       "source": [
-        "### Export with dynamic batch dimension\n",
-        "\n",
-        "_This is a new feature and will be included in pip installation after the 2.4 release, or if using `torch_xla` nightly. Once PyTorch/XLA 2.4 is released, this will be converted into a running example. Using the nightly `torch` and `torch_xla` will likely lead to notebook failures in the meantime._\n",
+        "_Tip:_\n",
         "\n",
-        "Dynamic batch dimensions can be specified as a part of the initial `torch.export` step. The FX Graph's symint information is used to export to dynamic StableHLO.\n",
+        "_Dynamic batch dimensions can be specified as a part of the initial `torch.export` step._\n",
         "\n",
-        "In this example, we specify that dim 0 of the sample input is dynamic, which propagates shape using a `tensor<?x3x224x224xf32>`.\n",
-        "\n",
-        "```python\n",
-        "from torch.export import Dim\n",
-        "\n",
-        "# Create a dynamic batch size, for the first dimension of the input\n",
-        "batch = Dim(\"batch\", max=15)\n",
-        "dynamic_shapes = ({0: batch},)\n",
-        "dynamic_export = export(resnet18, sample_input, dynamic_shapes=dynamic_shapes)\n",
-        "dynamic_stablehlo = exported_program_to_stablehlo(dynamic_export)\n",
-        "print(dynamic_stablehlo.get_stablehlo_text('forward')[0:5000],\"\\n...\")\n",
-        "```"
+        "_`torch_xla` doesn't yet support exporting models with dynamic shapes to StableHLO, so we recommend using the experimental library [`torch_xla2`](https://github.com/pytorch/xla/tree/master/experimental/torch_xla2) for this. It has a similar API for exporting, converts PyTorch models to JAX prior to exporting to StableHLO, and has high opset coverage along with great dynamic shape support._"
```

Comment:
`torch_xla` support for exporting dynamic models is limited, for these cases we recommend using `torch_xla2` (link). This lowering path leverages JAX for lowering to StableHLO, and has high opset coverage with much broader support for exported programs with dynamic shapes.

Reply:
Updated, thanks.